play-ebean/src/main/java/play/db/ebean/DefaultEbeanConfig.java

```diff
@@ -24,15 +24,24 @@
 /** Ebean server configuration. */
 @Singleton
 public class DefaultEbeanConfig implements EbeanConfig {
+  private final Boolean ddlGenerate;
```

Comment:
Let's rename this variable (and the config key, etc.) to
```suggestion
  private final Boolean generateEvolutionsScripts;
```